.github/ISSUE_TEMPLATE/bug_report_v2.yaml

```diff
@@ -0,0 +1,85 @@
+name: 🐞 Bug Report V2
+description: File a bug report
+title: '[Bug]: '
+body:
+  - type: markdown
+    id: before-you-start
+    attributes:
+      value: |
+        *[READ THIS]:* to evaluate if you are in the right place?
+        - For issues or feature requests related to __the code in this repository__, file a Github issue.
+        - If this is a __feature request__, make sure the issue title starts with "FR:".
+        - For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/) with the firebase tag.
+        - For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk) google group.
+        - For help troubleshooting your application that does not fall under one of the above categories, reach out to the personalized [Firebase support channel](https://firebase.google.com/support/).
+  - type: input
+    id: operating-system
+    attributes:
+      label: Operating System
+      description: Describe your operating system
+      placeholder: ex. iOS 16.4
+    validations:
+      required: true
+  - type: input
+    id: browser-version
+    attributes:
+      label: Browser Version
+      description: Describe your browser version
+      placeholder: ex. Safari/604.1
+    validations:
+      required: true
+  - type: input
+    id: firebase-sdk-version
+    attributes:
+      label: Firebase SDK Version
+      description: Describe your Firebase SDK Version
+      placeholder: ex. 9.16.0
+    validations:
+      required: true
+  - type: input
+    id: firebase-sdk-product
+    attributes:
+      label: Firebase SDK Product
+      description: Describe your Firebase SDK Product
+      placeholder: ex. Auth
+    validations:
+      required: true
+  - type: textarea
+    id: framework
+    attributes:
+      label: Describe the framework
+      description: Describe the framework your app is built with
+      placeholder: ex. React Native using ExpoGo
+    validations:
+      required: true
+  - type: markdown
+    id: describe-your-problem
+    attributes:
+      value: |
+        'Describe the problem'
```

Comment:
This probably should be required?

Reply:
This is just a header. I'll probably remove it